.env.example

```
@@ -0,0 +1,13 @@
+# with credentials for a dedicated empty Turso database.
+TURSO_DATABASE_URL=file:dogfood.db
+TURSO_AUTH_TOKEN=
+
+DURABLERUN_DOGFOOD_QUEUE=dogfood
+DURABLERUN_DOGFOOD_KEY=ref-journal-v1
+DURABLERUN_DOGFOOD_REPOSITORY=ejc3/durablerun
+DURABLERUN_DOGFOOD_REF=main
+DURABLERUN_DOGFOOD_CYCLES=15
+DURABLERUN_DOGFOOD_INTERVAL_SECONDS=43200
+DURABLERUN_DOGFOOD_LEASE_SECONDS=30
+DURABLERUN_DOGFOOD_FAULT=none
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DURABLERUN_DOGFOOD_FAULT key should go before the DURABLERUN_DOGFOOD_INTERVAL_SECONDS key

(UnorderedKey)